<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadWriteEEPROM</class>
 <widget class="QMainWindow" name="ReadWriteEEPROM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadWriteEEPROM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="logView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>311</width>
      <height>391</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>391</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="readWriteView">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>691</width>
      <height>391</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>391</height>
     </size>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLogsButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>650</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearReadWriteButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Clear Read/Write</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>1002</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelComPort">
           <property name="text">
            <string>Com Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comPorts"/>
         </item>
         <item>
          <widget class="QPushButton" name="detectComs">
           <property name="text">
            <string>Detect Coms</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="boudRate"/>
         </item>
         <item>
          <widget class="QComboBox" name="lineTermination"/>
         </item>
         <item>
          <widget class="QPushButton" name="connectButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectButton">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="statusLine"/>
         </item>
         <item>
          <widget class="QPushButton" name="readButton">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="writeButton">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelinFileName">
           <property name="text">
            <string>In file name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="inFileName"/>
         </item>
         <item>
          <widget class="QLabel" name="labelOutFile">
           <property name="text">
            <string>Out file name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outFileName"/>
         </item>
         <item>
          <widget class="QLabel" name="bufferSizelabel">
           <property name="text">
            <string>Buffer Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="bufferSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxLength">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setBuffer">
           <property name="text">
            <string>Set Buffer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelEepromType">
             <property name="text">
              <string>EPROM type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="eepromType"/>
           </item>
           <item>
            <widget class="QLabel" name="sizeLabel">
             <property name="text">
              <string>EPROM Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="epromSize"/>
           </item>
           <item>
            <widget class="QLabel" name="deviceAddressLabel">
             <property name="text">
              <string>Device Address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="deviceAddress"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="addresslabel">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="address"/>
         </item>
         <item>
          <widget class="QLabel" name="lengthLabel">
           <property name="text">
            <string>Read Length</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="length"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="dataToSend"/>
         </item>
         <item>
          <widget class="QGroupBox" name="writeBytes">
           <property name="title">
            <string>Write Bytes</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="allBytes">
              <property name="text">
               <string>All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="onlyByte">
              <property name="text">
               <string>&amp;1 Byte</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
